### PR TITLE
add pro chapter and a text about user management

### PR DESCRIPTION
**Pull request by michaelmattig**

### CODE REVIEW

**Comment by michaelmattig** [comment]
[Admin kann Rechte vergeben](https://trello.com/c/3H9dWVE0/1107-admin-kann-rechte-vergeben)

File: src/pro/intro.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Pro Features
+
+While much of Geo Engine's functionality is Open Source and freely usable, some parts are only available in the Pro version.
+To use the Pro version, you need to purchase a Pro license.
+You may, however, be eligible for a free academic license.
+Please contact as at [info@geoengine.de](mailto:info@geoengine.de?subject=Academic%20License) to request one.
```
Comment (jdroenner) [inline src/pro/intro.md:6]: as -> us

File: src/pro/users.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+The Pro version of Geo Engine includes a user management system.
+Users can either be _anonymous_ or _registered_.
+On the first startup, an admin user will be created.
+
+Geo Engine has a Role Based Access Control (RBAC) system.
+Users can have different roles and permissions on resources are granted to these roles.
+By default, they have a unique role for themselves and either the role `anonymous` or `registered`.
+The admin user has the role `admin`.
+
```
Comment (jdroenner) [inline src/pro/users.md:11]: can we add an example or is that to complex?

**Review by jdroenner — CHANGES_REQUESTED** [review]

File: src/pro/intro.md (hunk 1 above)
Comment (michaelmattig) [reply src/pro/intro.md]: 93eae05b63a135de3952bb2d0522ac366b2dc09a

File: src/pro/users.md (hunk 2 above)
Comment (michaelmattig) [reply src/pro/users.md:11]: what do you have in mind? Give an example for a resource and a role and some permissions and explain who has access and why? I can do that if you want.

Comment (jdroenner) [reply src/pro/users.md:11]: Exactly. Something like this so the reader can "see" what a permission is and what it does.

Comment (michaelmattig) [reply src/pro/users.md:11]: 147c86aa092324758bce9c9a96be08122a4bbb18

**Review by jdroenner — APPROVED** [review]

**Pull request merged**